schemas/skills/v2.2/skill-manifest.json

```diff
@@ -0,0 +1,372 @@
+            "type": "string",
+            "description": "The identifier for the skill manifest"
+        },
+        "name": {
+            "type": "string",
+            "description": "Name of the skill"
+        },
+        "version": {
+            "type": "string",
+            "description": "Skill version"
+        },
+        "description": {
+            "type": "string",
+            "description": "A human readable description for the skill"
+        },
+        "publisherName": {
+            "type": "string",
+            "description": "The name of the skill publisher"
+        },
+        "privacyUrl": {
+            "type": "string",
+            "format": "uri-reference",
+            "description": "The URL with the privacy description for the skill"
+        },
+        "copyright": {
+            "type": "string",
+            "description": "The copyright for the skill"
+        },
+        "license": {
+            "type": "string",
+            "description": "The license agreement for the skill"
+        },
+        "iconUrl": {
+            "type": "string",
+            "format": "uri-reference",
+            "description": "Optional icon to be shown for the skill"
+        },
+        "tags": {
+            "type": "array",
+            "uniqueItems": true,
+            "description": "An array of strings with tags for the skill",
+            "items": {
+                "type": "string"
+            }
+        },
+        "endpoints": {
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "description": "List of endpoints supported by the skill",
+            "items": {
+                "$ref": "#/definitions/endpoint"
+            }
+        },
+        "dispatchModels": {
+            "type": "object",
+            "description": "Language models and top level intents for the skill",
+            "properties": {
+                "languages": {
+                    "type": "object",
+                    "description": "List of languages supported by the skill",
+                    "minProperties": 1,
+                    "additionalProperties": {
+                        "type": "array",
+                        "minItems": 1,
+                        "uniqueItems": true,
+                        "description": "List of language models supported by the skill",
+                        "items": {
+                            "$ref": "#/definitions/languageModel"
+                        }
+                    }
+                },
+                "intents": {
+                    "type": "array",
+                    "uniqueItems": true,
+                    "description": "A list of the skill's top level intents that can be used by the consumer to filter out language files.",
+                    "items": {
+                        "type": "string"
+                    }
+                }
+            },
+            "additionalProperties": false
```

Comment:
Would you mind adding a "title" field to each of these alongside the descriptions? It can just be the same as the key but with proper capitalization and spacing (e.g. `Dispatch Models` for `dispatchModels`). I think these only need to be at the eleven top-level properties here.

Reply:
@beyackle, I added title to all the properties, can you please review before I merge?

Thanks